### ISSUE TITLE
Cursor jumps to random line when having delete button pressed for a longer time

### ISSUE THREAD

**Issue by PricacyPro0** [streetwriters/notesnook#6565]
### What happened?

When you have a bigger text block which you want to delete and you don't select the text and delete it with one click, but you keep the delete button pushed, then in some rare cases, the mous cursor will jump to a random spot a few lines below. 
Because you don't expect the cursor to jump to a random line below, you will delete some of it, if you don't release your delete button quick enough. 

It's random to wich line it jumps. 
It's random when this happens. 
It happens only once a week, not very often. But it happens from time  to time. 
If you try to delete something it happens one out of a hundred times. 
I have had this issue on Android (Graphenos), Linux (Fedora) and in different Notesnook versions. 

It's gonna be hard to solve this issue, because its hard to reproduce. But I wanted to let you guys know, because this has happened several times in a time period from over half a year. 

I personally don't think it's a user issue, because it happens if you just press the delete button long enough, while doing nothing else and not pressing any other key or space on your Touchscreen. 

It's a very very low priority issue. 

### Steps to reproduce the problem

Push the delete button for a long time 

### Version

3.0.17-6def76e-desktop

### Platform/OS

Linux, Android
